data.access.exercises/src/test/java/data/access/exercises/DatabaseUtility.java

```diff
@@ -0,0 +1,62 @@
+package data.access.exercises;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.*;
+
+public class DatabaseUtility implements DatabaseAccessor {
+
+	@Override
+	public String[] ExecuteSingleColumn(String sql) {
+		try (Connection myConn = DriverManager.getConnection("jdbc:mysql://localhost:3306/sakila","root","password");
+				Statement myStmt = myConn.createStatement();
+				ResultSet myRs = myStmt.executeQuery(sql);) {
+				
+				while(myRs.next()) {
+					System.out.println(myRs.getString("title") | myRs.getString("inventory_id"));
+				}
+			}
+			catch(Exception exc){
+				exc.printStackTrace();
+			}
+				return null;
```

Comment:
You're returning null. So effectively this utility doesn't do anything. Return the query results.